src/util/symbol_table_base.cpp

```diff
@@ -60,6 +60,16 @@ void symbol_table_baset::show(std::ostream &out) const
     out << symbols.at(name);
 }
 
+symbol_table_baset::const_iteratort symbol_table_baset::end() const
+{
+  return symbols.end();
```

Comment:
96bd3ae8e86ac8712a662e80d710c05a64b8442c `s/interate/iterate` in the commit message.

Also this commit should be removed; Current develop already has begin/end for `symbol_tablet`.